malware/nisseware-phishing/index.md

```diff
@@ -14,15 +14,270 @@ Nisseware - Phishing
 Malware
 
 ## Challenge Description:
+```text
+Vores indledende undersøgelser peger på, at gruppen kom ind via et målrettet phishingangreb, og vi har sikret en mistænkelig mail fra forurettede.
 
-Writeup will come...
+Analysér det vedhæftede dokument og afdæk, hvordan gruppen igangsatte næste stage.
+```
 
 ## Approach
 
-## Flag
+We're given [the mail](files/mail.eml) that originally allowed the malware to enter the victims system.
+
+Reading the mail is easy enough, it just base64 encoded text:
+```text
+Kære Alf,
+
+Jeg har færdiggjort og vedhæftet det detaljerede regnskab for vores salg i tredje kvartal af 2023.
+
+Regnskabet indeholder som altid et overblik over antal salg af hvert produkt, men vi har dette kvartal forsøgt også at opdele efter kategorier.
+Jeg tror, det vil give jer et solidt grundlag for at analysere markedstendenser, identificere styrker og svagheder i vores produktportefølje samt træffe informerede beslutninger om indkøb og egenproduktion i den kommende periode.
+
+Du må meget gerne videresende arket til dine kollegaer i afdelingen samt til afdelingen for forskning og produktion, hvis du finder det relevant.
+
+Jeg har tilføjet noget funktionalitet i filen, der sikrer, det kun kan åbnes på virksomhedens computere, så det ikke falder i forkerte hænder.
+Du skal derfor lige gemme regnearket, højre-klikke og vælge "Egenskaber" og så sætte flueben i "Fjern blokering", så skulle det hele virke 
+
+Jeg er til rådighed for yderligere forklaringer vedrørende oplysningerne i regnearket. Det er vigtigt for os at sikre, at I har alle nødvendige oplysninger for at træffe velinformerede beslutninger i forhold til jeres produktplanlægning.
+
+Fortsat god dag og arbejdslyst!
+
+Med venlig hilsen,
+
+Jytte Andersen
+Regnskabsafdelingen
+Legetøj"Я"Os
+```
+
+Nothing really important in the mail, basically just "please download and run this inconspicuous file, then forward it to all your colleagues", with surprisingly few spelling errors for a phishing mail.
+
+In the mail we've been given, there's also an attached file "[2023-Q3-regnskab.xlsm](files/2023-Q3-regnskab.xlsm)" (your virus/malware scanner should detect a threat - as far as I've been able to determine, its a false flag in this instance).
+
+The most common way to use excel sheets as malware transport, is via VBA scripting inside the sheet.
+
+I used [oletools](http://decalage.info/python/oletools) to extract any VBA script that might be inside the file:
+
+```text
+olevba 2023-Q3-regnskab.xlsm > vbadump
+```
+olevba finds the file "ThisWorkbook" in the spreadsheet, and extracts it.
+
+That gives us the follow two obfuscated sets of VBA:
+```VB
```

Comment:
VB doesn't appear to be recognized, so I would suggest something else like python to ensure highlighting!

```suggestion
```python
```